cognite_toolkit/_cdf_tk/data_classes/_config_yaml.py

```diff
@@ -87,27 +87,9 @@ def load(cls, data: dict[str, Any], build_name: str) -> Environment:
             name=build_name,
             project=data["project"],
             build_type=build_type,
-            selected=cls.load_selected(data.get("selected")),
+            selected=parse_user_selected_modules(data.get("selected")),
         )
 
-    @classmethod
-    def load_selected(cls, raw: list[str | Path] | None, organization_dir: Path | None = None) -> list[str | Path]:
-        # The type of raw path is set just to make mypy happy.
```

Comment:
Moved into utils and called `parse_user_selected_modules` to enable use independently of this data class